metrics/sample.go

```diff
@@ -394,7 +395,7 @@ func SampleVariance(values []int64) float64 {
 	return sum / float64(len(values))
 }
 
-// A uniform sample using Vitter's Algorithm R.
+// UniformSample uniform sample using Vitter's Algorithm R.
```

Comment:
// UniformSample represents a uniform sample using Vitter's Algorithm R.

Reply:
Thank you - updated.